inst/rmarkdown/templates/transmissibility/skeleton/skeleton.Rmd

```diff
@@ -346,17 +345,16 @@ dat_i %>%
 This graph shows the total number of cases per group:
 
 ```{r }
-total_cases <- dat %>%
-  tags_df() %>%
-  select(location, counts) %>%
-  group_by(location) %>%
-  summarise(cases = sum(counts)) %>%
-  mutate(location = fct_reorder(
-    .f = location,
+total_cases <- dat_i %>%
+  select(any_of(c(group_var, "count"))) %>%
+  group_by(.data[[group_var]]) %>%
+  summarise(cases = sum(count)) %>%
+  mutate(group_var := fct_reorder(
+    .f = .data[[group_var]],
     .x = cases
   ))
 
-ggplot(total_cases, aes(x = cases, y = location)) +
+ggplot(total_cases, aes(x = cases, y = group_var)) +
```

Comment:
so it's location instead of group_var because this is the linelist tag? If we have linelist tags, couldn't we use them directly instead of creating the objects date_var and group_var? or is this needed for data that are already aggregated?

Reply:
This is needed because the grouping variable may not always be `location`. It could be `gender` for example.